lib/widget/markdown.js

```diff
@@ -25,37 +23,36 @@ function Markdown(options) {
 
 Markdown.prototype = Object.create(Box.prototype);
 
-Markdown.prototype.setOptions = function(style) {
```

Comment:
so now this is invalid because there is no setOptions function:
```
const markdown = require('blessed-contrib').markdown()
markdown.setOptions({}) // fails
```

Reply:
Fixed. Also keep the other method evalStyle(). To be sincere, I doubt that thing ever worked, and if so we should document it or typing it or test it. Best regards! Keep it up!